Documentation/guides/BuildProcess.md

```diff
@@ -343,6 +343,9 @@ when packaging Release applications.
     ItemGroup contains default profile(s). It can be overriden by
     removing the existing one(s) and adding your own AOT profiles.
 
+    The profiles can also be added with `AndroidAotProfiles` property,
```

Comment:
`$(AndroidAotProfiles)` should be listed as a separate property, not grouped with `AndroidEnableProfiledAot`, in part because `AndroidEnableProfiledAot` was added to XA 9.4 while `$(AndroidAotProfiles)` will be added in XA 10.1/10.2 (10.1 if we CP this to d16-4).

Reply:
Which will place it far away from the `AndroidEnableProfiledAot` entry :-( It is property though, so I will make a it a separate entry.

I might write some separate piece of doc on how to use the profiled AOT from command line I guess.